docker-compose.yml

```diff
@@ -32,6 +31,49 @@ services:
     depends_on:
       admin_backend_db:
         condition: service_healthy
-
+  pgadmin:
+    image: dpage/pgadmin4
+    container_name: pgadmin4
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - 8000:80
+    # This has to have world & group access denied to work
+    entrypoint: /bin/sh -c "chmod 600 /pgpass; /entrypoint.sh;"
+    configs:
+      - source: servers.json
+        target: /pgadmin4/servers.json
+      - source: pgpass
+        target: /pgpass
+    environment:
+      # This deavctivates the login screen
+      PGADMIN_CONFIG_SERVER_MODE: "False"
+      PGADMIN_CONFIG_MASTER_PASSWORD_REQUIRED: "False"
+    volumes:
+      - pgadmin-data:/var/lib/pgadmin
+    # This is just nice to know as if it fails, it fails quietly
+    healthcheck:
+      test: [CMD, wget, -O, "-", http://localhost:80/misc/ping]
+configs:
+  pgpass:
+    content: "admin_backend_db:5432:*:${POSTGRES_USER}:${POSTGRES_PASSWORD}"
```

Comment:
Should we be using `PGHOST` for this? What if someone changes the value in the .env file?

Reply:
I thought hardcoding this to what's in the `Dockerfile` made sense as `localhost` wouldn't work here as the networking is happening between the services specified in this file.